tensorflow/core/util/BUILD

```diff
@@ -484,7 +484,7 @@ cc_library(
 cc_library(
     name = "port",
     srcs = ["port.cc"],
-    hdrs = ["port.h"],
+    hdrs = ["port.h", "onednn_env_vars.h"],
```

Comment:
I don't think we need to add "onednn_env_vars.h" here. This target should just depend on `:onednn_env_vars`.

Reply:
Thanks for your comment, without including "onednn_env_vars.h", it was not getting built successfully"

Reply:
Could you please try adding `deps = [":onednn_env_vars"]` instead?

Reply:
Thanks for your suggestion, moved it as a dependency